cluster/gce/config-default.sh

```diff
@@ -27,7 +27,7 @@ MINION_DISK_SIZE=100GB
 # for containervm image, so that gcloud will expand this
 # to the latest supported image.
 IMAGE=container-vm-v20150317
-IMAGE_PROJECT=google-containers
+IMAGE_PROJECT=google_containers
```

Comment:
To clarify, gcr.io is special in interpretting '_' as '-' in the project id portion of the image tag, which it does to support old docker versions. In actuality, GCP doesn't allow underscores in project names, which why gcr.io can be certain that `project-name` and `project_name` are the same thing.